substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -1379,6 +1388,34 @@ where
 		self.view_store
 			.listener
 			.add_view_aggregated_stream(view.at.hash, view_aggregated_stream.boxed());
+
+		view
+	}
+
+	/// Builds and updates a new view.
+	///
+	/// This functio uses [`Self::build_new_view`] to create or clone new view.
+	///
+	/// The new view will be updated with transactions from the tree_route and the mempool, all
+	/// required events will be triggered, it will be inserted to the view store (respecting all
+	/// pre-insertion actions).
+	async fn build_and_update_view(
+		&self,
+		origin_view: Option<Arc<View<ChainApi>>>,
+		at: &HashAndNumber<Block>,
+		tree_route: &TreeRoute<Block>,
+	) -> Option<Arc<View<ChainApi>>> {
+		let start = Instant::now();
+		debug!(
+			target: LOG_TARGET,
+			?at,
+			origin_view_at = ?origin_view.as_ref().map(|v| v.at.clone()),
+			?tree_route,
+			"build_new_view"
```

Comment:
```suggestion
			"build_and_update_view"
```

Same for the debug at the end.